Remotion/Web/Development.WebTesting/HostingStrategies/Configuration/HostingConfiguration.cs

```diff
@@ -43,31 +43,31 @@ public class HostingConfiguration : IHostingConfiguration
             { "Docker", typeof(DockerHostingStrategy) },
         };
 
-    private readonly ProviderSettings _hostingProviderSettings;
+    private readonly IWebTestHostingSettings _webTestHostingSettings;
     private TimeSpan _verifyWebApplicationStartedTimeout;
 
-    public HostingConfiguration ([NotNull] WebTestConfigurationSection webTestConfigurationSection, [NotNull] ITestSiteLayoutConfiguration testSiteLayoutConfiguration)
+    public HostingConfiguration ([NotNull] IWebTestSettings webTestSettings, [NotNull] ITestSiteLayoutConfiguration testSiteLayoutConfiguration)
     {
-      ArgumentUtility.CheckNotNull("webTestConfigurationSection", webTestConfigurationSection);
+      ArgumentUtility.CheckNotNull("webTestSettings", webTestSettings);
       ArgumentUtility.CheckNotNull("testSiteLayoutConfiguration", testSiteLayoutConfiguration);
 
-      _hostingProviderSettings = webTestConfigurationSection.HostingProviderSettings;
-      _verifyWebApplicationStartedTimeout = webTestConfigurationSection.VerifyWebApplicationStartedTimeout;
+      _webTestHostingSettings = webTestSettings.Hosting;
```

Comment:
We are already in the context of the web tests, which makes the `webTest` prefix redundant. I would suggest using `hostingSettings` as field name here.